pywbem/_recorder.py

```diff
@@ -523,89 +525,170 @@ class LogOperationRecorder(BaseOperationRecorder):
 
     All logging calls are at the debug level.
     """
-    def __init__(self, max_log_entry_size=None):
+    def __init__(self, conn_id=None, detail_levels=None):
         """
         Parameters:
 
-          max_log_entry_size (:term:`integer`):
+          conn_id ():)
+            If None, string that represents an id for a particular connection
```

Comment:
I assume this should have been: "If not None ..."?

Reply:
fixed DONE